src/services/item/plugins/app/appAction/index.ts

```diff
@@ -19,6 +21,8 @@ const plugin: FastifyPluginAsync = async (fastify) => {
   fastify.register(async function (fastify) {
     fastify.addHook('preHandler', fastify.verifyBearerAuth as preHandlerHookHandler);
 
+    if (WEBSOCKETS_PLUGIN) fastify.register(appActionsWsHooks, { appActionService });
```

Comment:
Add `{ ... }`

Reply:
Sorry, I don't get it? Can you give be an example?